apps/wallet/src/ui/app/staking/validators/SelectValidatorCard.tsx

```diff
@@ -34,15 +54,28 @@ export function SelectValidatorCard() {
         setSelectedValidator((state) => (state !== address ? address : null));
     };
 
+    const handleSortByKey = (key: 'name' | 'apy') => {
+        setSortKey(key);
+        setSortAscending(!sortAscending);
```

Comment:
Why do we always flip the asc / desc order when changing keys? Seems like we should just change the sort key if it changes, and only change asc / desc when re-clicking a selected key.